[DOCS] Improve Trusted Apps "wildcard"/"matches" field documentation (#1469)
elastic/security-docs: docs/management/admin/trusted-apps.asciidoc

+

* `Operator`: Select an operator to define the condition:
   ** `is`: Must be exactly equal to `Value`. This operation is required for the `Hash` and `Signature` field types.   

>    ** `matches`: Can include wildcards in `Value`, such as `C:\path\*\app.exe`. This option is only available for the `Path` field type. Available wildcards are `?` (match any one character) and `*` (match zero to many characters).

Reviewer: ```suggestion
   ** `matches`: Can include wildcards in `Value`, such as `C:\path\*\app.exe`. This option is only available for the `Path` field type. Available wildcards are `?` (match one character) and `*` (match zero or more characters).
```
